test/extended/util/framework.go

```diff
@@ -235,7 +236,7 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface,
 			}
 		}
 	}
-	return fmt.Errorf("the deploy did not finish within 3 minutes")
+	return fmt.Errorf("the deploy did not finish within %v minutes", maxWaitingTime)
```

Comment:
>As it happens, @gabemontero already caught and fixed this anyway (which is why this PR is now needing a rebase).

As far I can see Gabe has added name to the message while I'm fixing hard-coded time.